src/node.cc

```diff
@@ -1159,6 +1153,29 @@ Maybe<bool> ProcessEmitDeprecationWarning(Environment* env,
                                    deprecation_code);
 }
 
+static void OnMessage(Local<Message> message, Local<Value> error) {
+  Isolate* isolate = Isolate::GetCurrent();
+  Environment* env = Environment::GetCurrent(isolate);
```

Comment:
Can we move this line into the inner block, and return (i.e. silence the warning) if `env == nullptr`?